### ISSUE TITLE
SwapPanel throws ValueError

### ISSUE THREAD

**Issue by Acidham** [raguay/SwapPanels#4]
Not sure if this is related to #3 but I experiencing this error with SwapPanel 2.0. Is it required to apply #3 workaround for my own?

Command 'SwapPanel' raised error.

```
Traceback (most recent call last):
  File "fman/impl/plugins/command_registry.py", line 128, in _execute_command
  File "/Users/jjung/Library/Application Support/fman/Plugins/Third-party/SwapPanels/swappanels/__init__.py", line 47, in __call__
    lpane.set_path(rpane_path, _setRightPaneSelections)
  File "fman/__init__.py", line 130, in set_path
  File "fman/impl/widgets.py", line 122, in set_location
  File "fman/impl/model/__init__.py", line 48, in set_location
  File "fman/impl/model/__init__.py", line 45, in set_location
  File "fman/impl/model/__init__.py", line 76, in _set_location
  File "/Users/jjung/Library/Application Support/fman/Plugins/Third-party/SwapPanels/swappanels/__init__.py", line 36, in _setRightPaneSelections
    lpane.place_cursor_at(rpane_cursor)
  File "fman/__init__.py", line 115, in place_cursor_at
  File "fman/impl/util/qt/thread.py", line 11, in result
  File "fman/impl/util/qt/thread.py", line 81, in run_in_thread
  File "fman/impl/util/qt/thread.py", line 106, in result
  File "fman/impl/util/qt/thread.py", line 94, in __call__
  File "fman/impl/widgets.py", line 127, in place_cursor_at
  File "fman/impl/view/__init__.py", line 124, in place_cursor_at
  File "fman/impl/model/__init__.py", line 147, in find
  File "fman/impl/model/model.py", line 327, in find
ValueError: 'file:///Users/jjung/.bash_sessions' is not in list
```

**Comment by raguay** [comment]
The fix from #3 has been implemented and seems to be working fine. You might have an older version. Delete the plugin and then re-install it to get the latest version.

**Comment by Acidham** [comment]
Works thx for quick turn around!
